.circleci/config.yml

```diff
@@ -38,39 +19,19 @@ workflows:
           username_envar: "QUAY_USERNAME"
           password_envar: "QUAY_PASSWORD"
           requires:
-            - build
+            - go-build
           filters:
             tags:
               only: /^v.*/
 
       - architect/push-to-docker:
-          name: push-cluster-operator-to-aliyun-pr
```

Comment:
I made it to push to all registries all the time.

Reply:
Why should it be changed within this PR?

Reply:
Ah yeah it was weird. I guess someone removed hold step but not merge the jobs. All good 👍